### ISSUE TITLE
Feature Request: [Launchpad] Add context on what "Launch site" means

### ISSUE THREAD

**Issue by liviopv** [Automattic/wp-calypso#76947]
### What

We should have some context on what clicking to "Launch a site" actually means

### Why

Launching a site can sound too big of a change for a lot of customers because they are unaware that it can be reverted by setting the site to Public or Coming Soon.

By adding context on what that change represents, we can make customers more confident, which should lead to an increase in launched sites.many

### How

Linking to the [Privacy Settings](https://wordpress.com/support/settings/privacy-settings/) doc below the "Launch Site" button or in a tooltip might be a good first step

![lHiwxKJCaS.png](https://github.com/Automattic/wp-calypso/assets/14153300/ee0d3589-8e26-4f44-913f-873131531ea2)

**Comment by cuemarie** [comment]
📌 ACTIONS
- Feature request kept
- CC: @Automattic/serenity

**Comment by nelsonec87** [comment]
@nuriapenya , What do you think would be the ideal copy?

**Comment by nuriapenya** [comment]
Thank @nelsonec87, for the ping, and @liviopv, for raising this issue 🙏🏽  As I see this was created a while ago (15 May), I would like to take the chance to dig a bit more into this topic:

* Have you collected customer feedback since sharing this issue? Either from the CTA in particular or this page, in general. 
* From your experience, would changing the copy or adding an icon be enough? Or it could also be that the tasks displayed on this page are not enough to "do the jump" and launch. 

Thanks in advance!

For reference, in terms of copy, we refer to `Launch` in different areas:
<img width="1067" alt="Screenshot 2023-10-04 at 12 10 55" src="https://github.com/Automattic/wp-calypso/assets/7931304/3c090eaa-fddc-4032-b031-61dd58133729">

This sentence could help a lot, I think:
`Your site is private and only visible to you. When you're ready, launch your site to make it public.`

<img width="1439" alt="Screenshot 2023-10-04 at 12 11 07" src="https://github.com/Automattic/wp-calypso/assets/7931304/dfb178e2-8e5f-4222-8b41-cab1d3255029">

**Comment by andres-blanco** [comment]
Hi @liviopv pinging you again about this issue, specifically to know if you have an answer for this questions that Nuria raised:
* Have you collected customer feedback since sharing this issue? Either from the CTA in particular or this page, in general.
* From your experience, would changing the copy or adding an icon be enough? Or it could also be that the tasks displayed on this page are not enough to "do the jump" and launch.

Thanks :)

**Comment by markbiek** [comment]
Hi @liviopv. I sent you a Slack DM but I also wanted to reach out here again. We're hoping you may have some information about the questions above?

**Comment by liviopv** [comment]
Sorry I missed the ping!

> Have you collected customer feedback since sharing this issue? Either from the CTA in particular or this page, in general.

No, I haven't collected feedback intentionally, but anecdotal experience from interacting with customers every day indicates that a lot of customers (especially low-code) don't understand the nuances of Site Privacy.

This sometimes leads to situations where they will make all the individual posts/pages on a site private or drafts just to avoid having them "live" before they are ready, just to give an example.

People also don't understand that they are easily reversible and that they have different options for site visibility. For instance, for a lot of customers who might "skip to dashboard", a great option could be to switch the site to "Hidden from search engines" so they can share it with other people to get feedback before making it live, but the Launchpad flow doesn't surface that as an option.

From my perspective, Launchpad should be a tool to make people feel successful and knowledgeable for completing a certain task, but by putting a big milestone like "Launch site" as the only option at the end of it, we might be blocking their progression.

In video game terms, "Launch site" feels like an extremely hard boss at the end of the first area, and the game doesn't give any hints that the player can take another route or lower the difficulty if they don't feel ready to take on the challenge.

> From your experience, would changing the copy or adding an icon be enough? Or it could also be that the tasks displayed on this page are not enough to "do the jump" and launch.

I'm not opposed to "Launch site" as a term in this context, but I believe we could provide context (whether through a copy below the button, or a tooltip) and/or expose a link to Site Privacy to "customize your site's visibility" as an alternative to launch or exit the dashboard. It doesn't need to be all or nothing.

**Comment by markbiek** [comment]
@nuriapenya We'll plan on updating the copy based on https://github.com/Automattic/wp-calypso/issues/76947#issuecomment-1746072523.

**Comment by nuriapenya** [comment]
Yeah, let's try. Also now the completed tasks do not have the strikethrough, making it easier for the users to re-interact with them.

@markbiek let me know when you've got a link to check/test :) and thanks for following up!

**Comment by richtabor** [comment]
Let's pause work on this, as it's being phased out for an [in-WordPress launchpad](https://github.com/Automattic/wp-calypso/issues/98245). However, I do think it's important to make launching more meaningful, so I've started [this.](https://github.com/Automattic/wp-calypso/issues/98860).
